pkg/csi/cinder/driver.go

```diff
@@ -31,7 +31,7 @@ const (
 )
 
 var (
-	version = "0.3.0"
+	version = "1.0.0"
```

Comment:
Do we maintain always the same versioning according to CSI spec ? or could we add alpha, beta etc to the same to indicate the stability etc?

Reply:
guess keep with CSI spec will make it easier to sync and understand......

Reply:
keeping it as it is